doc/install.md

```diff
@@ -233,7 +237,13 @@ from Python.
 To ensure that PyGMT and its dependencies are installed correctly, run the following
 in your Python interpreter:
 
-```python
+```{code-cell} ipython
+---
+tags: [remove-stdout]
+mystnb:
+  image: center
+---
```

Comment:
```
---
tags: [remove-stdout]
mystnb:
  image: center
---
```
These lines are cell-level metadata for controlling the format of the output. Usually it's not necessary. Here `remove-stdout` is necessary to hide the long output from `pygmt.show_versions()`.